corehq/apps/userreports/pillow.py

```diff
@@ -243,73 +307,16 @@ def get_filtered_configs(self, configs=None):
             elif self.ucr_division:
                 configs = _filter_by_hash(configs, self.ucr_division)
 
-            configs = _filter_domains_to_skip(configs)
+            if self.is_dedicated_migration_process:
+                # Non-migration processes always fetch configs by domain,
+                # so do not need to discard configs for missing domains.
+                # This is only useful for static data sources, which do not change.
+                configs = _filter_domains_to_skip(configs)
+
             configs = _filter_invalid_config(configs)
 
         return configs
 
-    def _do_bootstrap(self, configs=None):
-        configs = self.get_filtered_configs(configs)
-        if not configs:
-            pillow_logging.warning("UCR pillow has no configs to process")
-
-        self.table_adapters_by_domain = defaultdict(list)
-
-        for config in configs:
-            self.table_adapters_by_domain[config.domain].append(
-                _get_indicator_adapter_for_pillow(config)
-            )
-
-    @property
-    def relevant_domains(self):
-        return set(self.table_adapters_by_domain)
-
-    def get_adapters(self, domain):
-        return list(self.table_adapters_by_domain.get(domain, []))
-
-    def get_all_adapters(self):
-        return [
-            adapter
-            for adapter_list in self.table_adapters_by_domain.values()
-            for adapter in adapter_list
-        ]
-
-    def remove_adapter(self, domain, adapter):
-        self.table_adapters_by_domain[domain].remove(adapter)
-
-    def _update_modified_since(self, timestamp):
-        """
-        Find any data sources that have been modified since the last time this was bootstrapped
-        and update the in-memory references.
-        """
-        new_data_sources = [
-            source
-            for provider in self.data_source_providers
-            for source in provider.get_data_sources_modified_since(timestamp)
-        ]
-        filtered_data_sources = self.get_filtered_configs(new_data_sources)
-        invalid_data_sources = {ds._id for ds in new_data_sources} - {ds._id for ds in filtered_data_sources}
```

Comment:
This never worked because the values in `invalid_data_sources` were expected to be _data source config_ objects, not ids.